src/chainlit/frontend/src/components/organisms/inputs/inputStateHandler.tsx

```diff
@@ -2,23 +2,35 @@ import { Box, FormControl, FormHelperText } from '@mui/material';
 
 import InputLabel from 'components/molecules/inputLabel';
 
-type InputStateHandlerProps = {
+import { IInput } from 'types/Input';
+
+export type InputStateHandlerProps = {
   children: React.ReactNode;
-  description?: string;
-  hasError?: boolean;
-  id: string;
-  label?: string;
-  tooltip?: string;
-};
+} & IInput;
 
 export default function InputStateHandler(
   props: InputStateHandlerProps
 ): JSX.Element {
-  const { children, description, hasError, id, label, tooltip } = props;
+  const {
+    children,
+    description,
+    hasError,
+    id,
+    label,
+    notificationsCount,
+    tooltip
+  } = props;
 
   return (
     <Box width="100%">
-      {label ? <InputLabel id={id} label={label} tooltip={tooltip} /> : null}
+      {label ? (
```

Comment:
Minor suggestion, I like to use condition && component instead of condition ? component : null

Reply:
It's a better practice to render null to be sure react will not render unwanted value. https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx